cli/pcluster/dcv/connect.py

```diff
@@ -71,31 +100,18 @@ def dcv_connect(args):
             error(
                 "Something went wrong during DCV connection. Please manually execute the command:\n{0}\n"
                 "If the problem persists, please check the logs in the /var/log/parallelcluster/ folder "
-                "of the master instance and submit an issue {1}.".format(cmd, PCLUSTER_ISSUES_LINK)
+                "of the master instance and submit an issue {1}".format(ssh_cmd, PCLUSTER_ISSUES_LINK)
             )
 
     except sub.CalledProcessError as e:
         if "{0}: No such file or directory".format(DCV_CONNECT_SCRIPT) in e.output:
             error(
                 "The cluster {0} has been created with an old version of ParallelCluster "
-                "without the DCV support.".format(args.cluster_name)
+                "without the DCV support.".format(cluster_name)
```

Comment:
"Without DCV support"

Reply:
"without" is a continuation of the previous row ;)